app/helpers/podcast_planner_helper.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module PodcastPlannerHelper
+  PERIODIC_WEEKS = I18n.t([:every_one, :every_two, :every_three, :every_four], scope: ".podcast_planner.helper.period_options").freeze
```

Comment:
I think these need to be like `scope: [:podcast_planner, :helper, :period_options]`